src/main/java/com/zendesk/maxwell/replication/BinlogConnectorEventListener.java

```diff
@@ -1,36 +1,38 @@
 package com.zendesk.maxwell.replication;
 
-import java.util.concurrent.BlockingQueue;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicBoolean;
-
 import com.codahale.metrics.Timer;
 import com.github.shyiko.mysql.binlog.BinaryLogClient;
 import com.github.shyiko.mysql.binlog.event.Event;
 import com.github.shyiko.mysql.binlog.event.EventType;
 import com.github.shyiko.mysql.binlog.event.GtidEventData;
-
+import com.github.shyiko.mysql.binlog.event.QueryEventData;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+
 class BinlogConnectorEventListener implements BinaryLogClient.EventListener,
 					      BinaryLogClient.LifecycleListener {
 	private static final Logger LOGGER = LoggerFactory.getLogger(BinlogConnectorEventListener.class);
 
 	private final BlockingQueue<BinlogConnectorEvent> queue;
 	private Timer queueTimer;
+	private Timer mysqlTxExecutionTimer;
```

Comment:
It's nice to have those private parameters `final`, but not mandatory. Can setup IDE to do so.